feature-wallet-impl/build.gradle

```diff
@@ -100,4 +100,6 @@ dependencies {
     implementation zXingEmbeddedDep
 
     implementation insetterDep
+
+    implementation navigationFragmentDep
```

Comment:
Navigation should only be in app module. Feature modules use XXXRouter interface to pefrorm navigation